src/inspector_socket.h

```diff
@@ -58,7 +59,7 @@ class InspectorSocket {
   struct http_parsing_state_s* http_parsing_state;
   struct ws_state_s* ws_state;
   std::vector<char> buffer;
-  uv_tcp_t client;
+  uv_tcp_t client;  // XXX(sam) this is a server, why is it called "client"?
```

Comment:
tcp/socket - both sound good. Ultimately, I believe this field name came from libuv API uv_accept that calls the connection socket "client" as opposed to "server" that is a listening socket.